File: sol/solution/8 Kyu/559f3123e66a7204f000009f/Solution.cs
/*
Solution
*/
using System;
using System.Linq;
public class Kata
{
  public static string ArrowFunc(int[] arr)
  {
    return string.Join("", arr.Select(i => Tools.FromCharCode(i)).ToArray());
  }
}





/*
Sample Tests
*/
using NUnit.Framework;
using System;

[TestFixture]
public class Test
{
  [Test]
  public static void FixedTests()
  {
    Assert.That(Kata.ArrowFunc(new int[]{84,101,115,116}), Is.EqualTo("Test"));
  }
}

File: sol/solution/8 Kyu/559f3123e66a7204f000009f/Solution.js
/*
Solution
*/
const arrowFunc = (arr) => arr.map(num => String.fromCharCode(num)).join('');





/*
Sample Tests
*/
const { assert } = require('chai');

describe('Example tests', () => {
  it('should convert numbers to letters', () => {
    tester([84,101,115,116], 'Test');
    tester([70,85,83,32,82,79,72,32,68,65,72], 'FUS ROH DAH');
  });
});

function tester(arr, expected) {
  const actual = arrowFunc(arr);
  assert.strictEqual(actual, expected, `Testing for arr = ${JSON.stringify(arr)}\n\n`);
}